libs/community/langchain_community/embeddings/text2vec.py

```diff
@@ -0,0 +1,52 @@
+"""Wrapper around text2vec embedding models."""
+
+from typing import Any, List
+
+from langchain_core.embeddings import Embeddings
+from langchain_core.pydantic_v1 import BaseModel
+from text2vec import SentenceModel
+
+
+class Text2vecEmbeddings(BaseModel, Embeddings):
+    """text2vec embedding models.
+
+    Install text2vec first, run 'pip install -U text2vec'.
```

Comment:
1) Could you provide a reference to the github repository for text2vec? So maintainers can inspect whether this is an OK dependency to add to the code base?
2) langchain_community uses optional imports exclusively -- imports should be in the local scope

for example: https://github.com/langchain-ai/langchain/blob/main/libs/community/langchain_community/chat_message_histories/redis.py#L28-L28

Reply:
The gitbub repository for text2vec is : 
https://github.com/shibing624/text2vec

The link of your example returns a 404 error.

Reply:
What changes should I make to be merged? It is the second time that I write a little thing to pull the request. I fell first the lint check where I find nothing to change or improve in my code.

Reply:
OK great! I didn't realize it was https://github.com/shibing624/text2vec. Yeah we can add it